deployments/config/user-dashboard.yml

```diff
@@ -8,6 +8,22 @@ http:
       tls:
         certResolver: letsencryptresolver
 
+    user-dashboard-www-https:
+      entryPoints:
+        - "websecure"
+      rule: 'Host(`www.admin.{{ env "DOMAIN_NAME" }}`) && PathPrefix(`/`)'
```

Comment:
This should allow `https://www.admin.gasteropo.de` to be redirected to `https://admin.gasteropo.de`, just like for the main website.

Reply:
After testing it it seems like we fail to generate the right certificates for `www.admin.gasteropo.de`:

```
Unable to obtain ACME certificate for domains error="unable to generate a certificate for the domains [www.admin.gasteropo.de]: error: one or more domains had a problem:\n[www.admin.gasteropo.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.admin.gasteropo.de - check that a DNS record exists for this domain; DNS problem: NXDOMAIN looking up AAAA for www.admin.gasteropo.de - check that a DNS record exists for this domain
```